Tfschema client fails on initializing aliased provider names

### Description
Bug fix #65 has introduced a regression where resources that specify an aliased provider name fail.
The root cause is that the aliased provider name is sent to `tfschema` client init which isn't aware of these customized provider names.

### Error stack
```
panic: Failed to NewClient: Failed to find plugin: aws.custom. Plugin binary was not found in any of the following directories: [tf-files/hunters/test1, /Users/yaronyarimi/dev/terratag, tf-files/hunters/test1/terraform.d/plugins/darwin_amd64, tf-files/hunters/test1/.terraform/plugins/darwin_amd64, /Users/yaronyarimi/.terraform.d/plugins, /Users/yaronyarimi/.terraform.d/plugins/darwin_amd64, /Users/yaronyarimi/go/bin]

goroutine 1 [running]:
github.com/env0/terratag/errors.PanicOnError(...)
        /Users/yaronyarimi/dev/terratag/errors/errors.go:10
github.com/env0/terratag/tfschema.getClient(0xc00079a521, 0xa, 0x7ffeefbff838, 0x16, 0xc000e5e540, 0xc000e5e690)
        /Users/yaronyarimi/dev/terratag/tfschema/tfschema.go:92 +0x257
github.com/env0/terratag/tfschema.IsTaggable(0x7ffeefbff838, 0x16, 0x0, 0xc0004a5130, 0xc000e9dfb0, 0xc000e5e030, 0xc000e5e540, 0xc000e5e690, 0xc000d73d40, 0xc000d73da0, ...)
        /Users/yaronyarimi/dev/terratag/tfschema/tfschema.go:25 +0x160
main.tagFileResources(0xc00044b440, 0x38, 0x7ffeefbff838, 0x16, 0x7ffeefbff855, 0x77, 0xc, 0x1, 0x0, 0x0, ...)
        /Users/yaronyarimi/dev/terratag/main.go:96 +0x40f
main.tagDirectoryResources(0x7ffeefbff838, 0x16, 0xc000456000, 0x5c, 0x80, 0x7ffeefbff855, 0x77, 0x1, 0xc, 0x1, ...)
        /Users/yaronyarimi/dev/terratag/main.go:70 +0x22c
main.Terratag(0x7ffeefbff855, 0x77, 0x7ffeefbff838, 0x16, 0x0, 0x0, 0x10001)
        /Users/yaronyarimi/dev/terratag/main.go:58 +0xe1
main.main()
        /Users/yaronyarimi/dev/terratag/main.go:31 +0x97
```

### Example

```hcl
provider "aws" {
  region  = "us-east-1"
  alias   = "custom"
  version = "~> 3.0"
}

resource "aws_s3_bucket" "bucket" {
  provider = aws.custom

  bucket = "my-tf-test-bucket"
  acl    = "private"
}
```

--- a/test/fixture/terraform_13/git_issue_66_aws_aliased_provider/expected/main.terratag.tf
+++ b/test/fixture/terraform_13/git_issue_66_aws_aliased_provider/expected/main.terratag.tf
@@ -1,0 +1,17 @@
+provider "aws" {
+  region  = "us-east-1"
+  alias   = "custom"
+  version = "~> 3.0"
+}
+
+resource "aws_s3_bucket" "bucket" {
+  provider = aws.custom
+
+  bucket = "my-tf-test-bucket"
+  acl    = "private"
+  tags   = local.terratag_added_main
+}
+locals {
+  terratag_added_main = {"env0_environment_id"="40907eff-cf7c-419a-8694-e1c6bf1d1168","env0_project_id"="43fd4ff1-8d37-4d9d-ac97-295bd850bf94"}
+}
+

--- a/test/fixture/terraform_13/git_issue_66_aws_aliased_provider/expected/main.tf.bak
+++ b/test/fixture/terraform_13/git_issue_66_aws_aliased_provider/expected/main.tf.bak
@@ -1,0 +1,12 @@
+provider "aws" {
+  region  = "us-east-1"
+  alias   = "custom"
+  version = "~> 3.0"
+}
+
+resource "aws_s3_bucket" "bucket" {
+  provider = aws.custom
+
+  bucket = "my-tf-test-bucket"
+  acl    = "private"
+}

--- a/test/fixture/terraform_13/git_issue_66_aws_aliased_provider/input/main.tf
+++ b/test/fixture/terraform_13/git_issue_66_aws_aliased_provider/input/main.tf
@@ -1,0 +1,12 @@
+provider "aws" {
+  region  = "us-east-1"
+  alias   = "custom"
+  version = "~> 3.0"
+}
+
+resource "aws_s3_bucket" "bucket" {
+  provider = aws.custom
+
+  bucket = "my-tf-test-bucket"
+  acl    = "private"
+}

--- a/tfschema/tfschema.go
+++ b/tfschema/tfschema.go
@@ -2,6 +2,7 @@ package tfschema
 
 import (
 	"fmt"
+	"github.com/thoas/go-funk"
 	"log"
 	"strings"
 
@@ -15,6 +16,8 @@ import (
 )
 
 var providerToClientMap = map[string]tfschema.Client{}
+
+var customSupportedProviderNames = [...]string{"google-beta"}
 
 func IsTaggable(dir string, resource hclwrite.Block) bool {
 	var isTaggable bool
@@ -54,21 +57,26 @@ type TfSchemaAttribute struct {
 	Type string
 }
 
+func extractProviderNameFromResourceType(resourceType string) (string, error) {
+	s := strings.SplitN(resourceType, "_", 2)
+	if len(s) < 2 {
+		return "", fmt.Errorf("Failed to detect a provider name: %s", resourceType)
+	}
+	return s[0], nil
+}
+
 func detectProviderName(resource hclwrite.Block) (string, error) {
 	providerAttribute := resource.Body().GetAttribute("provider")
 
-	if providerAttribute == nil {
-		resourceType := resource.Labels()[0]
-		s := strings.SplitN(resourceType, "_", 2)
-		if len(s) < 2 {
-			return "", fmt.Errorf("Failed to detect a provider name: %s", resourceType)
-		}
-		return s[0], nil
-	} else {
+	if providerAttribute != nil {
 		providerTokens := providerAttribute.Expr().BuildTokens(hclwrite.Tokens{})
 		providerName := strings.Trim(string(providerTokens.Bytes()), "\" ")
-		return providerName, nil
+		if funk.Contains(customSupportedProviderNames, providerName) {
+			return providerName, nil
+		}
 	}
+
+	return extractProviderNameFromResourceType(terraform.GetResourceType(resource))
 }
 
 func getClient(providerName string, dir string) tfschema.Client {

